web/views.py

```
@@ -6929,3 +6933,207 @@ def delete_post(request, post_id):
+                    target_date=target_date,
+                    priority=2  # Default to medium priority
+                )
+
+        messages.success(request, 'Study plan created successfully!')
+        return redirect('study_planner')
+
+    return render(request, 'study_planner/create_plan.html')
+
+
+@login_required
+def study_plan_detail(request, plan_id):
```

Ruff (0.8.2)

Missing return type annotation for public function `study_plan_detail`

(ANN201)